wordpress-mobile/WordPress-iOS
# Prime for Push: alert primer

Refs #9185

This replaces the system push notification alert during login with a priming fancy alert that appears _after_ login. The associated analytics events are included. It also adds the feature flag that the rest of development for #9185 will use.

![9185 alert animation](https://user-images.githubusercontent.com/517257/39341881-6340f190-4992-11e8-872f-146649f6b011.gif)

### Notes:
- The order of the buttons is reversed from the mockup, to be consistent with our other fancy alerts. @SylvesterWilmott approved this change.
- Sylvester is also going to review the copy for this alert, it may get minor changes before merging.

To test:
- delete existing WP copy and install a fresh build
- login
- at the end of login, notice that the push notification primer appears
- also notice the events being tracked in the console

cc: @frosty I made some changes to the `FancyAlertViewController` to allow tracks events when appearing. Let me know if you have any concerns?
